### PR TITLE
modify modules config

### PR DESCRIPTION
**Pull request by aMichaelPoernomo**
### Issues

- no type & lint error, but scss modules returned an empty object

### CODE REVIEW

File: ui/webpack.config.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = {
             loader: 'css-loader',
             options: {
               importLoaders: 1,
-              modules: { auto: true }
+              modules: true
```
Comment (aStanleyLiang) [inline ui/webpack.config.js:32]: it's weird that the original one is the same as the one in the document https://webpack.js.org/loaders/css-loader/

Any test have you done after apply this change to check the result ?

Comment (aMichaelPoernomo) [reply ui/webpack.config.js:32]: Here, I attached the images that happened on my local.

Screen Shot 2020-12-25 at 5 42 57 PM
Screen Shot 2020-12-25 at 5 43 38 PM
Screen Shot 2020-12-25 at 5 46 07 PM
Screen Shot 2020-12-25 at 5 46 22 PM
Screen Shot 2020-12-25 at 5 46 46 PM

Comment (aStanleyLiang) [reply ui/webpack.config.js:32]: Ok, I found it in the wepback document: https://webpack.js.org/loaders/css-loader/#modules

Your is correct one. I think this is a documentation by webpack

截圖 2020-12-28 上午9 46 24
截圖 2020-12-28 上午9 46 15

**Review by aStanleyLiang — APPROVED** [review]

**Review by aYishinHo — APPROVED** [review]

**Comment by aStanleyLiang** [comment]
@aMichaelPoernomo  I suggest to merge this despite @aEricLiao  doesn't review this yet. @aEricLiao ensure you know the changes Michael made.

**Pull request merged**